### PR TITLE
Support Gradle 3.4

### PR DESCRIPTION
**Pull request by ben-manes**
Work in progress. Doesn't handle resolution strategies properly, so may not be the correct approach.

@jochenberger

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.4-rc-2-bin.zip
```
Comment (jochenberger) [inline gradle/wrapper/gradle-wrapper.properties]: You don't have to update the wrapper version to test against new Gradle versions. TestKit takes care of forking a build with the specified release.

Comment (jochenberger) [inline gradle/wrapper/gradle-wrapper.properties]: In fact, I think we should not make this change in this PR, This just affects the Gradle version that is used to *build* the plugin. https://github.com/ben-manes/gradle-versions-plugin/pull/131/commits/ba4d411e955a4dff40a40c05c78585fed82fc515 adds 3.4-rc-2 to the versions to test against.

Comment (ben-manes) [inline gradle/wrapper/gradle-wrapper.properties]: Sure, we can drop this. I was manually testing example build scripts

**Comment by jochenberger** [comment]
That doesn't seem to be right, it also breaks Gradle 2.0 and 1.12 (and probably others that we don't test against).

**Pull request closed without merging**